backend/handler/tests/conftest.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import alembic.config
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from config.config_loader import ConfigLoader
+from models.platform import Platform
+from models.rom import Rom
+
+engine = create_engine(ConfigLoader.get_db_engine(), pool_pre_ping=True)
+session = sessionmaker(bind=engine, expire_on_commit=False)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_database():
+    alembic.config.main(argv=["upgrade", "head"])
+
+
+@pytest.fixture(autouse=True)
+def clear_database():
+    with session.begin() as s:
+        s.query(Platform).delete(synchronize_session="evaluate")
+        s.query(Rom).delete(synchronize_session="evaluate")
```

Comment:
This is the best way I've found (so far) to migrate a fresh test database and clear it after the tests run.